### ISSUE TITLE
Multiple Notices

### ISSUE THREAD

**Issue by david-24** [Ghostff/Debug#2]
If i have multiple NOTICES it will catch only one.

**Comment by Ghostff** [comment]
Yes when using `BittrDbug::PRETTIFY` errors are display one at a time. I will consider implementing  multiple error output support in my next push.
